src/InstagramClone.Api/Startup.cs

```diff
@@ -13,26 +15,47 @@ public static WebApplication InitialIzeApp(string[] args)
             Configure(app);
             return app;
         }
-       
+
 
         private static void ConfigureService(WebApplicationBuilder builder)
         {
             
+            builder.Services.AddControllers(config =>
+            {
+                config.RespectBrowserAcceptHeader = true;
+                config.ReturnHttpNotAcceptable = true;
+            }).AddXmlDataContractSerializerFormatters();
+
+             builder.Services.AddControllers();
+            builder.Services.AddCors();
+
             builder.Services.AddDbContext<InstagramCloneDbContext>(options =>
             {
                 options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnectionString"));
 
-               
+
             });
-          
+
         }
 
         private static void Configure(WebApplication app)
         {
+            app.UseHttpsRedirection();
+
+            app.UseRouting();
+
+            
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+            });
```

Comment:
please add a line of comment for each line explaining what each does. A better thing will be a line to explain and an external url to the documentation for each

Reply:
Ok, Done.